lib/matplotlib/figure.py

```diff
@@ -182,10 +182,12 @@ def __init__(self, left=None, bottom=None, right=None, top=None,
             The top of the subplots of the figure
 
         *wspace* : 0.2
```

Comment:
They're from a time before numpydoc; since numpydoc formats parameters in bold anyway, these have been made redundant. Just no-one's gone through to fix them yet.